rfcs/formats/network/iin_method.md

```diff
@@ -0,0 +1,264 @@
+The DID create method of the IIN registry must authenticate the a Network DID creation request differently from other DIDs. For a Network DID, the IIN first validates if the DID Document has the requisite properties: `networkParticipants`, and a verificationMethod of type `GroupMultiSig`. Additionally, the request must contain the signatures from all participants of the network authenticating it.
+
+The authentication process is carried out by validating signature from each participant in the `networkParticipants` as follows:
+
+1. The participant DID is obtained from `networkParticipants`.
+   - Example: `did:iin:<iin_name>:<network_participant_1>`
+2. From the `GroupMultiSig` verification, which verification method for the participant will be used is determined.
+   - Example: `did:iin:<iin_name>:<network_participant_1>#key1` indicates that key1 will be used which is specified by the fragment at the end of the URI.
+3. The DID Document of the participant is obtained by resolving the DID, and the required verification method is obtained.
+   - Example:
+   ```json
+    {​
+      "id": "did:iin:<iin_name>:<network_participant_1>",​
+      ...​
+      "verificationMethod": [
+        {
+          "id": "did:iin:<iin_name>:<network_participant_1>#key1",
+          "type": "Bls12381G2Key2020",
+          "controller": "did:iin:<iin_name>:<network_participant_1>",
+          "publicKeyBase58": "25ETdUZDVnME6yYuAMjFRCnCPcDmYQcoZDcZuXAfeMhXPvjZg35QmZ7uctBcovA69YDM3Jf7s5BHo4u1y89nY6mHiji8yphZ4AMm4iNCRh35edSg76Dkasu3MY2VS9LnuaVQ",
+
+        }]
+    "assertionMethod": ["did:example:org1#key1"],
+    "authentication": ["did:example:org1#key2"]​
+    }​
+   ```
+
+4. From `singatures` in Network DID creation request, the participant is validated with the help of the verificationMethod obtained in the previous step.
+
+
+Once each participant's signature is validated, the Network DID is registered in the IIN registry.
+
+Since the IIN registry is a decentralized registry, the validation of a Network DID creation request is carried out by multiple peers so that consensus is reached, and the DID is committed in the registry.
+
+### Read
+
+
+
+Given the DID in the format `did:iin:<iin_name>:<network_name>` or `did:iin:<iin_name>:<participant_name>` (example `did:iinindy:tradelens`), the corresponding DID Document can be resolved.
+
+Example Read request:
+
+```
+{
+    "id": "did:iin:ibmiin:tradelens.network"
+ }
+ ```
+
+
+
+### Update
+
+#### Preparing Network DID updation request
+
+Updating a Network DID requires the request to be authenticated against the `GroupMultiSig` verification method's `updatePolicy`.
+
+Similar to the Network DID creation request, the updation request has a NetworkDIDDocument and signatures. NetworkDIDDocument contains the updated Network DID document and signatures is a set of signatures satisfying the  `updatePolicy` in the `GroupMultiSig` verification method of the existing DID document, as well as the `networkParticipants`.
```

Comment:
NetworkDIDDocument in some kind of quotes. Probably "signatures" too?
"document, and signatures is...."
There's a double space before `updatePolicy`.
"as well as the `networkParticipants`": couldn't parse this; how does it fit the sentence?